frontend/src/views/Timeline.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div id="app">
+    <v-app id="inspire">
```

Comment:
remove line 2 and 3. You are getting errors because your branch is not updated. Then try to convert to pug